pkg/syncer/spec/mutators/deployment.go

```diff
@@ -61,18 +55,18 @@ func (dm *DeploymentMutator) GVR() schema.GroupVersionResource {
 	}
 }
 
-func NewDeploymentMutator(upstreamURL *url.URL, secretLister ListSecretFunc, syncTargetLogicalClusterName logicalcluster.Name,
+func NewDeploymentMutator(upstreamURL *url.URL, secretLister ListSecretFunc, serviceLister listerscorev1.ServiceLister,
```

Comment:
is it possible to use a `ListSecretFunc` following the same pattern as what was used for secrets ?

Reply:
Got it